src/modules/common/components/bars/topBar/hardwareWallet.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Tooltip from 'src/theme/Tooltip';
+import { useTranslation } from 'react-i18next';
+import Icon from 'src/theme/Icon';
+import { HW_CONNECTION_STATUS } from '@libs/hwServer/constants';
+import styles from './hardwareWallet.css';
+
+const Status = ({ status }) => (
+  <div className={`${styles.statusWrapper} ${styles[status]}`}>
+    <b>{status}</b>
+  </div>
+);
+
+const HardwareWallet = () => {
+  const { t } = useTranslation();
+
+  const status = HW_CONNECTION_STATUS.DISCONNECTED;
```

Comment:
And this will come from the `currentDevice.status`?

Reply:
I have added a todo here this would be replaced when the useHWstatus hook has been integrated